dockers/docker-database/database_config.json.j2

```diff
@@ -83,6 +83,11 @@
             "id" : 12,
             "separator": "|",
             "instance" : "redis_chassis"
+        },        
+        "CHASSIS_STATE_DB" : {
+            "id" : 13,
+            "separator": "|",
+            "instance" : "redis-chassis"
```

Comment:
it is underscore here - `redis_chassis`

Reply:
also can you change GLOBAL_STATE_DB to CHASSIS_STATE_DB in commit message?

Reply:
Fixed.